markdown/stories/run_jobs_on_grid.md

```diff
@@ -14,34 +14,27 @@ on a metworx instance to make sure there are no errors based on the environment.
 
 Test | Test Name | Count
 -----|-----------|-------
-bbi_sge_test.go| <ul><li>TestBabylonCompletesSGEExecution</li><li>TestBabylonCompletesParallelSGEExecution</li></ul> |2
+bbi_sge_test.go| TestBabylonCompletesSGEExecution | 1
+bbi_sge_test.go| TestBabylonCompletesParallelSGEExecution | 1
 
 ##### Automated Test Explanations
-* TestBabylonCompletesSGEExecution : For automated testing, a fake `qsub` binary is created by the test that just
-exits with a code of 0 (normally). This way, Babylon in SGE mode doesn't execute automatically, but creates the `grid.sh`
-that would be executed by a remote SGE worker. The test then locates the shell script and executes it to verify functionality.
-Each scenario is evaluated to see:
-    * NonMem creates a `grid.sh` script that would be executed by a remote worker.
+* TestBabylonCompletesSGEExecution : Using a drone-exec runner against an Metworx workflow, drone runs these tests
+against an actual grid engine cluster. Babylon operates in normal SGE for this mode and after we have verified
+the execution on the SGE host has completed, we assert the following:
     * NonMem completes execution: The generated LST file is evaluated for a completion time
     * NonMem creates output files: The test makes sure that certain expected files (*.cpu, *.grd) exist in the output
     directory after completion of Nonmem
     * After executing `grid.sh`, Babylon creates a shell script: In each output directory should be a shell script <`modelname`>.sh which is
-    executed to perform the actual work.
+      executed to perform the actual work.
+
 * TestBabylonCompletesParallelSGEExecution : Similar to the above with the exception that parallel mode is requested for execution.
 This means that not only does the output file contain the grid reference, but also secondarily that the node count on the grid.
     * NonMem creates a `grid.sh` script that would be executed by a remote worker.
     * NonMem completes execution: The generated LST file is evaluated for a completion time
+    * NonMem references a Parafile: Looks through the generated LST file to locate the PARAFILE reference, indicating
+      the execution happened in parallel
     * NonMem creates output files: The test makes sure that certain expected files (*.cpu, *.grd) exist in the output
-    directory after completion of Nonmem
+      directory after completion of Nonmem
     * After executing `grid.sh`, Babylon creates a shell script: In each output directory should be a shell script <`modelname`>.sh which is
-    executed to perform the actual work.
-
-##### Manual Tests
-
-Test | Test Name | Count
```

Comment:
We removed the "manual tests" section from the markdown used to generate validation docs now that everything can be completely automated